src/app/examples/icon-example/icon-example.component.html

```diff
@@ -1,36 +1,44 @@
 <h2>Default Icons</h2>
-<kirby-icon name="add" title="add"></kirby-icon>
-<kirby-icon name="cog" title="cog"></kirby-icon>
-<kirby-icon name="swap" title="swap"></kirby-icon>
-<kirby-icon name="move" title="move"></kirby-icon>
-<kirby-icon name="log-out" title="log-out"></kirby-icon>
-<kirby-icon name="more" title="more"></kirby-icon>
-<kirby-icon name="arrow-back" title="arrow-back"></kirby-icon>
-<kirby-icon name="help" title="help"></kirby-icon>
-<kirby-icon name="close" title="close"></kirby-icon>
-<kirby-icon name="attach" title="attach"></kirby-icon>
-<kirby-icon name="search" title="search"></kirby-icon>
-<kirby-icon name="checkbox" title="checkbox"></kirby-icon>
-<kirby-icon name="checkbox-outline" title="checkbox-outline"></kirby-icon>
-<kirby-icon name="menu" title="menu"></kirby-icon>
-<kirby-icon name="person" title="person"></kirby-icon>
+<p class="note">*If icon is <strong>NOT</strong> found, 'cog' will be used as default icon</p>
+
+<div class="icon-item-container" *ngFor="let icon of icons">
+    <div class="icon-item-inner-container">
+        <kirby-icon [name]="icon.name" size="md" title="md"></kirby-icon>
```

Comment:
Remove size and use icon.name as title:
```suggestion
        <kirby-icon [name]="icon.name" [title]="icon.name"></kirby-icon>
```